docs/providers/mysql/index.rst

```diff
@@ -1,6 +1,17 @@
 MySQL (Coming Soon)
 ===================
 
-We are working with the MySQL team to make a MySQL provider available. Please direct any questions about this provider, including the release timeline, to the `MySQL Project <http://forums.mysql.com/list.php?38>`_.
+We are working with the MySQL team to make a MySQL provider available. 
 
+The official MySQL provider announcement with initial support for .NET Core and EF Core is available at http://insidemysql.com/mysql-connectornet-7-0-4-m3-has-been-released/
+
+Please direct any questions about this provider, including the release timeline, to the `MySQL Project <http://forums.mysql.com/list.php?38>`_.
+
+Commercial MySQL Provider(s)
+----------------------------
 DevArt dotConnect does offer a paid provider for MySQL, see :doc:`/providers/devart/index` for details.
+
+
+Other Open Source MySQL Provider(s)
+-----------------------------------
+*  *Pomelo Foundation*: There is an open sourced and free mysql provider for ef core from *Pomelo Foundation* at https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql
```

Comment:
Casing! There is also: https://www.nuget.org/packages/SapientGuardian.EntityFrameworkCore.MySql/